Cargo.lock

```diff
@@ -247,7 +247,7 @@ checksum = "afb2e1c3ee07430c2cf76151675e583e0f19985fa6efae47d6848a3e2c824f85"
 
 [[package]]
 name = "pipe-rename"
-version = "1.1.4"
+version = "1.2.0"
```

Comment:
Hmm I'm wondering how you can have your own crate as a dependency in the Cargo.lock, what is this magic?